### PR TITLE
EVG-2141 add system-out and system-err to junit logs

### PR DESCRIPTION
**Pull request by john-m-liu**
I went with a low-risk approach since the previous 2 attempts to refactor the code broke things

### CODE REVIEW

File: command/results_xunit.go [hunk 1]
```diff
@@ -140,6 +140,7 @@ func (c *xunitResults) parseAndUploadResults(ctx context.Context, conf *model.Ta
 				// logs are only created when a test case does not succeed
 				test, log := tc.toModelTestResultAndLog(conf.Task)
 				if log != nil {
+					log.Lines = append(log.Lines, "system-out:", suite.SysOut, "system-err:", suite.SysErr)
```
Comment (tychoish) [inline command/results_xunit.go]: should we only append if the values are populated? 

can we add some integration tests?

Comment (john-m-liu) [reply command/results_xunit.go]: added an integration test

Comment (tychoish) [reply command/results_xunit.go]: what about only adding these lines if they're populated?

Comment (john-m-liu) [reply command/results_xunit.go]: Done

**Pull request merged**